wgpu-core/src/device/resource.rs

```diff
@@ -1086,6 +1086,17 @@ impl Device {
                         .saturating_sub(desc.range.base_array_layer),
                 });
 
+        let resolved_usage = if desc.usage.is_empty() {
```

Comment:
This should probably be `desc.usage == TextureUsages::FROM_PARENT`

Reply:
Both are same, but your seems more explicit so I will take that.